src/setupProxy.js

```diff
@@ -0,0 +1,25 @@
+const { createProxyMiddleware } = require("http-proxy-middleware");
+
+// NOTE: Set your proxy with .env (use .env.sample)
+// otherwise it will fallback to localserver:8000
+
+// backround link on manual proxy config:
+// https://create-react-app.dev/docs/proxying-api-requests-in-development/#configuring-the-proxy-manually
+
+var backendserver = process.env.REACT_APP_BACKEND;
+if (
+  process.env.REACT_APP_BACKEND === null ||
```

Comment:
```suggestion
 !process.env.REACT_APP_BACKEND
```

Reply:
second line can then be deleted after suggestion since the falsy check includes empty strings, `undefined` & `null`

https://developer.mozilla.org/en-US/docs/Glossary/Falsy